portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -23,10 +25,34 @@
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
+  private List<String> quotes;
```

Comment:
nit: You could use an ImmutableList and inline everything:

```java
private final static ImmutableList<String> QUOTES = ImmutableList.of("A ship in port is safe, ...", "They told me computers ...", ...);
```

Reply:
When I tried to do "import com.google.common.collect.ImmutableList;" I got an error. Is there something else I have to import to use ImmutableList?

Reply:
Could you paste the error as well? If it's something like:
Main.java:2: error: package com.google.common.collect does not exist import

you may need to include Guava dependency in your pom.xml: 
<dependency>
  <groupId>com.google.guava</groupId>
  <artifactId>guava</artifactId>
  <version>11.0.2</version>
</dependency>

Reply:
That fixed it, thanks!